webui/react/src/components/Avatar.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+
+import Avatar from './Avatar';
+
+const initName = { initials: 'BB', name: 'Bugs Bunny' };
+
+jest.mock('antd', () => {
+  const antd = jest.requireActual('antd');
+
+  /** We need to mock Tooltip in order to override getPopupContainer to null. getPopupContainer
+   * sets the DOM container and if this prop is set, the popup div may not be available in the body
+   */
+  const Tooltip = (props) => {
+    return <antd.Tooltip {...props} getPopupContainer={null} mouseEnterDelay={0} />;
```

Comment:
**question:** looking at the TypeScript definition for `ToolTip` for prop `getPopupContainer`:
```
getPopupContainer?: (triggerNode: HTMLElement) => HTMLElement;
```
`null` is technically not a valid entry. Would it work to have it set to the following?
```
<ToolTip getPopupContainer={trigger => trigger} />
```
Where it gets added to the triggering element?

Reply:
Yeah, that works too. I'll change it to that so it's a little more explicit.